enneagramdate/enneagramdate
# Sam/login&signup

# Overview

## Task

Same as used in the feature branch


## Description
Login and signup pages without styling. In order to send user info to the backend as form data, you must upload an image (needed this for testing, it will be changed to always send as form data). Route protection and session persistence are WIP.

## Feature Validation Steps

0. Use the DB setup steps as mentioned in previous PRs
1. Click on the signup button on the home page and fill out the form.
2. Hitting go should take you to the recommendations page.
3. Refresh the app (but not the DB) and go to the login page.
4. Using your username and password from before should take you to the recommendations page.
